src/main/java/org/terasology/logic/inventory/StartingInventoryComponent.java

```diff
@@ -25,41 +26,37 @@
 
 /**
  * A class that allows you to specify a player's starting inventory easily.
- *
- * <p><strong>Note</strong> it is not possible to specify nested items (e.g. items in a chest). Stackable
- * <em>items</em> may have a quantity greater than their maxStackSize and they will be split into multiple
+ * <p>
+ * The default amount of items to add to the inventory is '1'. Not specifying a quantity explicitly will use this
+ * default value.
+ * <p>
+ * Stackable <em>items</em> may have a quantity greater than their maxStackSize and they will be split into multiple
  * stacks as required. However stackable <em>blocks</em> are limited to maximum stacks of 99, any excess
  * is ignored.</p>
  *
+ * <p><strong>Note</strong> it is not possible to specify nested items (e.g. items in a chest).
```

Comment:
According to line 71 it is? :thinking: Maybe I'm only misunderstanding this note, though?

Reply:
You are right! 🤦